src/mono/wasm/build/WasmApp.targets

```diff
@@ -66,7 +66,7 @@
                                               Defaults to false.
       - $(WasmAotProfilePath)               - Path to an AOT profile file.
       - $(WasmExceptionHandling)            - Enable support for the WASM Exception Handling feature.
-      - $(WasmSIMD)                         - Enable support for the WASM SIMD feature.
+      - $(WasmEnableSIMD)                         - Enable support for the WASM SIMD feature.
```

Comment:
I wanted to keep it similar to other feature props, like the `WasmExceptionHandling` above. Shouldn't we then change that too?